src/main/java/com/faforever/client/preferences/PreferencesService.java

```diff
@@ -178,8 +186,28 @@ private void readExistingFile(Path path) {
     try (Reader reader = Files.newBufferedReader(path, CHARSET)) {
       logger.debug("Reading preferences file {}", preferencesFilePath.toAbsolutePath());
       preferences = gson.fromJson(reader, Preferences.class);
-    } catch (IOException e) {
+    } catch (Exception e) {
       logger.warn("Preferences file " + path.toAbsolutePath() + " could not be read", e);
+      CountDownLatch waitForUser = new CountDownLatch(1);
+      Platform.runLater(() -> {
+        Alert errorReading = new Alert(AlertType.ERROR, "Error reading setting. Reset settings? ", ButtonType.YES, ButtonType.CANCEL);
```

Comment:
Notification service and internationalization are not loaded so we use those default error pop ups